modules/examples/todolist/client/src/main/scala/clients/TodoListClient.scala

```diff
@@ -36,4 +33,8 @@ trait TodoListClient[F[_]] {
 
   def remove(id: Int): F[Int]
 
-}
\ No newline at end of file
+}
+
+object TodoListClient {
+  trait Handler[G[_]] extends TodoListClient[G]
```

Comment:
Same as in [my other comment](https://github.com/frees-io/freestyle-rpc/pull/296/files#r192694425).